Libraries/libcar/Sources/AttributeList.cpp

```diff
@@ -44,10 +44,12 @@ void AttributeList::
 dump() const
 {
     for (auto const &entry : _values) {
+        constexpr const auto num_identifiers =
```

Comment:
`constexpr const` is redundant. I'd go for `const auto num_identifiers` because it won't make a difference in the generated code and easier to read, but `constexpr auto num_identifiers` is ok too if you prefer.

Reply:
Sure, changed to `constexpr auto`

Reply:
Thank you sir.